renderer/plugin.js

```diff
@@ -198,6 +201,14 @@ org.ekstep.contentrenderer.questionUnitPlugin = Plugin.extend({
    */
   getAudioIcon:function(path){
     return this.getAssetUrl(org.ekstep.pluginframework.pluginManager.resolvePluginResource(this._manifest.id, this._manifest.ver, path));
+  },
+  renderMath: function (event) {
+    setTimeout(function () {
+      jQuery('.math-text').each(function (index, element) {
```

Comment:
This is taking all the '.math-text' element always, even though when you want to render only one text element using event handler.

It has to render only specific element which has to render.